### DISCUSSION TITLE
Welcome to Agent-S Discussions!

### DISCUSSION THREAD

**Discussion by simularlyon** [simular-ai/Agent-S#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by myeongjaeking** [comment]
Hello, I'm a software engineering student. Thank you for creating such a great model! I'm reaching out because I have a question regarding this particular model.

I was considering implementing an AI agent using Episodic Memory—storing user experiences from specific websites or reservation systems—and employing ID-Grounding and OCR to train it on website HTML and button texts. Given this approach, I believe a search engine might not be necessary. Could you please clarify the purpose of Perplexica and why it is used in this context?

Thank you!

**Comment by perera2k4** [comment]
I have a question similar to @myeongjaeking's, about generating an automaton for automated frontend testing.
